data_quality_pipeline/src/made/data_pipeline/steps/unimodal_vision_filtering.py

```diff
@@ -62,7 +63,8 @@ def unimodal_vision_filtering(tar_files: list[str | Path], log_folder: Path, con
             apply_filters=config.infrastructure.apply_filters,
             parameters = {
                 "image_min_aspect_ratio": config.unimodal.image_min_aspect_ratio,
-                "image_max_aspect_ratio": config.unimodal.image_max_aspect_ratio
+                "image_max_aspect_ratio": config.unimodal.image_max_aspect_ratio,
+                "image_min_dimension": config.unimodal.image_min_dimension
```

Comment:
Is `image_min_dimension` related with the _text in image_ filter or is an additional unrelated feature of the _image size_ filter?